coordinator/src/main/scala/filodb.coordinator/queryengine2/QueryEngine.scala

```diff
@@ -344,14 +344,13 @@ class QueryEngine(dataset: Dataset,
     else { dataset.rowKeyIDs ++ ids }
   }
 
-  private def toRowKeyRange(rangeSelector: RangeSelector): RowKeyRange = {
+  private def toChunkScanMethod(rangeSelector: RangeSelector): ChunkScanMethod = {
     rangeSelector match {
-      case IntervalSelector(from, to) => RowKeyInterval(BinaryRecord(dataset, Seq(from)),
-                                                        BinaryRecord(dataset, Seq(to)))
-      case AllChunksSelector          => AllChunks
-      case EncodedChunksSelector      => EncodedChunks
-      case WriteBufferSelector        => WriteBuffers
-      case InMemoryChunksSelector     => InMemoryChunks
+      case IntervalSelector(from, to) => TimeRangeChunkScan(from, to)
+      case AllChunksSelector          => AllChunkScan
+      case EncodedChunksSelector      => ???
```

Comment:
Why leave this line here if not supported?

Reply:
It's for future support.  We'd like to be able to query just the encoded chunks for debugging.

Also re spark.  No it's not used.  I don't want to get rid of it as surely we will have to support some kind of ad hoc / Spark based querying of long term data, so it could still be useful in the future.

We should think about getting rid of the stress module though.  Or rewriting stress tests for current environment.